static/scripts.js

```diff
@@ -30,6 +32,14 @@ function initHotKeys() {
           searchInput.focus();
         }
       }
+
+      if (bodyEl.classList.contains('has-active-search-window')) {
+        if (key === 'Enter' || key === 'Return' || key === 13) {
+          event.preventDefault();
+          let firstresult = document.querySelectorAll("div#full-results a")[0];
+          firstresult.click();
+        }
```

Comment:
Since you're touching this rn, how hard would it be to allow the user to select results using the up-down arrow keys? Just a matter of catching those and storing/modifying an "index" state variable?

If it's trivial, would be great to have that added. If not, probably out of scope here. (I might already be guilty of scope-creeping this PR.)

Reply:
*shakes fist at `~palfun`*

Reply:
Thought for a sec. I do think it's a touch out of scope at the moment, since we're dealing with multiple arrays of results, and four new key events plus now tracking state (across search sessions on the same page as well). Would have to rethink search data structure for a sec and then test. Like it's not even really that hard, it's just that this is just meant to be a chore PR.

Reply:
Still sounds like good news to me. Very fair! (: